src/frontend/komponenter/Behandling/Vurdering/VurderingConfig.ts

```diff
@@ -0,0 +1,30 @@
+import { UnntakType, VilkårType } from '../Inngangsvilkår/vilkår';
+
+/**
+ * Gjør det mulig å splitte opp vurderinger i eks Medlemskap, Aleneomsorg, etc.
+ * Når man eks legger til en vurdering til medlemskap i VurderingConfig nå så kommer den opp automatisk
+ */
+export enum VilkårDel {
+    MEDLEMSKAP = 'MEDLEMSKAP',
+}
+
+export interface IVilkårConfig {
+    vilkårDel: VilkårDel;
+    vilkår: string;
+    unntak?: UnntakType[];
+}
+
+type IVurderingConfig = {
+    [key in VilkårType]: IVilkårConfig;
+};
+
+export const VurderingConfig: IVurderingConfig = {
+    FORUTGÅENDE_MEDLEMSKAP: {
+        vilkårDel: VilkårDel.MEDLEMSKAP,
+        vilkår: 'Vilkår for vurdering om utenlandsopphold er oppfylt',
```

Comment:
Denne teksten henger ikke helt på greip for meg. Burde det ikke stått noe á la "Vilkår for forutgående medlemskap er oppfylt"?

Reply:
Japp, vi må avklare dette med Mirja på nytt. Skal purre på dem idag igen.

Reply:
Vi fikser denne i en ny PR, når vi fått nye skisser